docs/operations/data_migration.md

```diff
@@ -78,55 +78,241 @@ An example script using this technique is in
 
 `--limitParse 100` 
 Directs the script to load only the first 100 entries into the table.  You can adjust this number as needed for testing purposes.  
 
 `--resetTable`
-This will delete all the data loaded into transtion_domain.  It is helpful if you want to see the entries reload from scratch or for clearing test data.
+This will delete all the data in transtion_domain.  It is helpful if you want to see the entries reload from scratch or for clearing test data.
 
 
 ### STEP 2: Transfer Transition Domain data into main Domain tables
 
 Now that we've loaded all the data into TransitionDomain, we need to update the main Domain and DomainInvitation tables with this information.  
-
 In the same terminal as used in STEP 1, run the command below; 
 (This will parse the data in TransitionDomain and either create a corresponding Domain object, OR, if a corresponding Domain already exists, it will update that Domain with the incoming status. It will also create DomainInvitation objects for each user associated with the domain):
 ```shell
-docker compose run -T app ./manage.py transfer_transition_domains_to_domains
+docker compose run -T app ./manage.py transfer_transition_domains_to_domains --debug
 ```
 
-**OPTIONAL COMMAND LINE ARGUMENTS**: 
+##### COMMAND LINE ARGUMENTS:
+
 `--debug`
 This will print out additional, detailed logs.
 
 `--limitParse 100` 
-Directs the script to load only the first 100 entries into the table.  You can adjust this number as needed for testing purposes.  
\ No newline at end of file
+Directs the script to load only the first 100 entries into the table.  You can adjust this number as needed for testing purposes.  
+
+### STEP 3: Send Domain invitations
+
+To send invitations for every transition domain in the transition domain table, execute the following command:
+```shell
+docker compose run -T app send_domain_invitations -s
+```
+
+### STEP 4: Test the results (Run the analyzer script)
+
+This script's main function is to scan the transition domain and domain tables for any anomalies.  It produces a simple report of missing or duplicate data.  NOTE: some missing data might be expected depending on the nature of our migrations so use best judgement when evaluating the results.
+
+#### OPTION 1 - ANALYZE ONLY
+
+To analyze our database without running migrations, execute the script without any optional arguments:
+```shell
+docker compose run -T app ./manage.py master_domain_migrations --debug
+```
+
+#### OPTION 2 - RUN MIGRATIONS FEATURE
+
+To run the migrations again (all above migration steps) before analyzing, execute the following command (read the documentation on the terminal arguments below.  Everything used by the migration scripts can also be passed into this script and will have the same effects).  NOTE: --debug and --prompt allow you to step through the migration process and exit it after each step if you need to.  It is recommended that you use these arguments when using the --runMigrations feature:
```

Comment:
I think we should keep the --runMigrations flag and instead we could force the prompts to execute unless disabled (for integration testing purposes).  This might be out of scope for the current ticket, so I will bring up to Alysia to include in a separate ticket.